src/index.ts

```diff
@@ -246,6 +255,24 @@ export class AcrolinxEndpoint {
     return getData<PlatformInformation>(this.getJsonFromPath('/api/v1/'));
   }
 
+  public async getAIEnabled(): Promise<AIEnabledInformation> {
+    return this.getJsonFromPath('/ai-service/api/v1/tenants/feature/ai-enabled');
+  }
+
+  public async getAIChatCompletion(prompt: string, count: number, issueInternalName: string): Promise<WriteResponse> {
```

Comment:
@JustinHorn I would recommend to create type of props needed for ChatCompletion

```
export type ChatRequest = {
  prompt: string;
  count: number;
  issueName: string
}
``` 

It will allow us extend interface in future without causing breaking change, and also avoids mixing arguments while passing